app/controllers/questionnaire_controller.rb

```diff
@@ -90,7 +121,8 @@ def questionnaire_params
   def questionnaire_create_params
     # TODO: change the below line to the following in rails 5.1:
     # params.permit(:response_id, content: {})
-    params.permit(:response_id, content: permit_recursive_params(params[:content]&.to_unsafe_h))
+    params.permit(:response_id, content: permit_recursive_params(params[:content]&.to_unsafe_h),
+                                stop_subscription: permit_recursive_params(params[:stop_subscription]&.to_unsafe_h))
```

Comment:
stop_subscription is een key/value hash net zoals content dat is. Ze zijn apart zodat je niet stop_subscription keys per ongeluk bij de antwoorden van een vragenlijst opslaat.